### PR TITLE
Updated CloneFromSnapshot CRD for restore

### PR DESCRIPTION
**Pull request by lintongj**
Created PR for CRD change specifically per Xing's request at https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/138#issuecomment-665866786

Signed-off-by: Lintong Jiang <lintongj@vmware.com>

### CODE REVIEW

File: hack/update-generated-crd-code.sh [hunk 1]
```diff
@@ -28,8 +28,9 @@ if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
   exit 1
 fi
 
-if [[ ! -d "${GOPATH}/src/sigs.k8s.io/controller-tools" ]]; then
-  echo "sigs.k8s.io/controller-tools missing from GOPATH"
+if ! command -v controller-gen > /dev/null; then
+  echo "controller-gen is missing"
+  echo "please retry after running the following command locally: go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4"
```
Comment (xing-yang) [inline hack/update-generated-crd-code.sh:33]: Can you try 0.3.0?

Comment (lintongj) [reply hack/update-generated-crd-code.sh:33]: Sure. Updated.

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/101/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/102/

**Pull request merged**